assets/scss/modules/_navigation.scss

```diff
@@ -147,3 +145,145 @@ a.back-to-top-link {
   }
 
 }
+
+/*
+Page Navigation
+
+A page navigation element with a previous and next link.
+
+Markup:
+<nav class="page-nav" role="navigation" aria-label="Page navigation">
+  <ul>
+    <li class="page-nav__item page-nav__item--previous">
+      <a class="page-nav__link page-nav__link-previous" title="Navigate to previous part" rel="prev" href="#">
```

Comment:
does this wrap ok in the code example in the comp lib? if it doesn't please format it so it is easy to read